x-pack/test/security_solution_endpoint_api_int/apis/resolver/entity_id.ts

```diff
@@ -38,26 +36,10 @@ export default function resolverAPIIntegrationTests({ getService }: FtrProviderC
       });
 
       it('excludes events that have an empty entity_id field', async () => {
-        // first lets get the _id of the document using the parent.process.entity_id
-        // then we'll use the API to search for that specific document
-        const res = await es.search<SearchResponse<Event>>({
```

Comment:
Clean up based on Rob's previous comments. The `_id` is now saved from the `bulk` response.